recipes-connectivity/connman/connman_%.bbappend

```diff
@@ -1,3 +1,5 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
 SRC_URI += "file://disable_connman_dns_proxy.patch"
+
+RDEPENDS += " connman-client"
```

Comment:
Don't you need `_${PN}` here?

And will this not cause a circular dependency since the ConnMan recipe has this: `RDEPENDS_${PN}-client ="${PN}"`

Reply:
Or does this actually work as is? Tested?

Reply:
you do

Reply:
circular dependency is handled fine

Reply:
Hum, yeah. I guess it does not matter that it is circular since it is run-time. OK, if this works it seems like a better solution than Pelagicore/meta-bistro#254 .